app/views/wiki/_form.html.erb

```diff
@@ -18,8 +17,18 @@
       <%= f.label :body %>
       <%= f.text_area :body, rows: '25', class: 'form-control'%>
     </div>
-
-    <%=f.submit "Post"%>
+    <% if locals[:collab] %>
```

Comment:
Try reducing to using `collab` instead of `locals[:collab]`. Partial templates shouldn't need to reference `locals`. Lets look at this further if its not working. Consider changing all `locals[:...]` calls to use the variable directly, e.g. `locals[:wiki]` becomes `wiki`.